packages/gestalt/src/Button.css

```diff
@@ -8,6 +8,7 @@
 
 .button {
   composes: borderBox from "./Layout.css";
+  border-radius: 8px;
```

Comment:
can we compose rounded from Borders.css here instead of adding more

Reply:
I was undoing this as closely as possible: https://github.com/pinterest/gestalt/pull/476/files#diff-0ae4272be4122c77370759def80050a3L11

Reply:
that one had a custom border radius of 4px, i think its best to reuse as much as we can now that its 8px and we have a class for that